x-pack/auditbeat/module/sysinfo/_meta/docs.asciidoc

```diff
@@ -0,0 +1,22 @@
+== Sysinfo Module
+
+The `sysinfo` module ... TODO.
```

Comment:
I was thinking to name it `system` to follow the Metricbeat and Filebeat tradition, but maybe that creates more confusion? I’m not sure.

Reply:
I think it would be OK to call it "system" if we avoid creating "metricsets" that have the same name as the existing ones in Metricbeat. We use nearly the exact same config between Auditbeat and Metricbeat so calling it the same thing could be confusing (even to us if we just glance at a config), but with unique names then it should be clear.

Reply:
I propose we merge it as is and let the module devs take ownership of naming.

Reply:
Agreed